migrations/kruize_local_ddl.sql

```diff
@@ -1,4 +1,5 @@
-create table IF NOT EXISTS kruize_datasources (version varchar(255), name varchar(255), provider varchar(255), serviceName varchar(255), namespace varchar(255), url varchar(255), authentication jsonb, primary key (name));
+create table IF NOT EXISTS kruize_authentication (id serial, authentication_type varchar(255), credentials jsonb, service_type varchar(255), primary key (id));
+create table IF NOT EXISTS kruize_datasources (version varchar(255), name varchar(255), provider varchar(255), serviceName varchar(255), namespace varchar(255), url varchar(255), authentication_id serial, FOREIGN KEY (authentication_id) REFERENCES kruize_authentication(id), primary key (name));
```

Comment:
add following columns
status  ->  connected , failed , 
last_check_date ->  timestamp when  last connection status happend
retry_count -> 
latest_result_message -> if connection failed

Reply:
We can do it in this PR or we can create a backlog and take up next rel

Reply:
Will take it up in a separate PR since this already contains a lot of file changes.
Have created this issue to track it: https://github.com/kruize/autotune/issues/1366